pyhanko/sign/general.py

```diff
@@ -365,8 +365,11 @@ def load_certs_from_pemder(cert_files):
         A generator producing :class:`.asn1crypto.x509.Certificate` objects.
     """
     for ca_chain_file in cert_files:
-        with open(ca_chain_file, 'rb') as f:
+        try:
+         with open(ca_chain_file, 'rb') as f:
             ca_chain_bytes = f.read()
+        except:
+            ca_chain_bytes=ca_chain_file.read()    
```

Comment:
Minor nitpick: add spaces around the `=`.

Reply:
Okie